workers/generic-worker/schemas/multiuser_posix.yml

```diff
@@ -247,6 +247,15 @@ properties:
           is one.
 
           Since: generic-worker 49.2.0
+      loopbackVideo:
+        type: boolean
+        title: Loopback Video device
+        description: |-
+          Video loopback device created using v4l2loopback.
+          A video device will be available to the task user
+          at `/dev/video0`.
```

Comment:
This is ok for a first pass, but i think eventually we'll need to make it configurable on the worker. Eventually we will want to support capacity > 1 on generic-worker workers, and then they will need to use unique paths.

Reply:
I can switch it to be a worker configuration that defaults to `/dev/video0` for now. We'll have to keep this in mind though as we move forward with the capacity > 1 on generic worker workers.